public/validators/main-albatross/NQ83 4MVH 53Q4 AL3B Q097 55GJ LUQ3 GSF0 85B7.json

```diff
@@ -0,0 +1,15 @@
QuMjQ3NSAtMi44MTI1IDQuMzc1IEMtNS4yNjIzMTM0OCA0Ljk1NDc1OTg1IC01LjI2MjMxMzQ4IDQuOTU0NzU5ODUgLTggNyBDLTguMzMgNi4zNCAtOC42NiA1LjY4IC05IDUgQy04LjM0IDUgLTcuNjggNSAtNyA1IEMtNi42NyA0LjAxIC02LjM0IDMuMDIgLTYgMiBDLTMuODU0MTI2MTYgMC42ODQ3ODcgLTIuNTI5ODY4OTUgMCAwIDAgWiAiIGZpbGw9IiM0NjJBMUEiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDMyMCw0NDcpIi8+CjxwYXRoIGQ9Ik0wIDAgQzAuMzMgMC42NiAwLjY2IDEuMzIgMSAyIEMwLjM0IDIgLTAuMzIgMiAtMSAyIEMtMS4yNDc1IDIuNTk4MTI1IC0xLjQ5NSAzLjE5NjI1IC0xLjc1IDMuODEyNSBDLTMuMTYzNzAwMTIgNi4yODY0NzUyMSAtNC42NTA1NTM4MyA3LjQzMzcwMjU1IC03IDkgQy01LjY0OTc4MTc0IDQuNTgxMTAzODcgLTMuNzM4ODQzMzIgMi42NTY1NDY1NyAwIDAgWiAiIGZpbGw9IiM1QjQzMzUiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDk2LDE1NCkiLz4KPHBhdGggZD0iTTAgMCBDMyAzIDMgMyAzLjE4NzUgNi4xODc1IEMzLjA5NDY4NzUgNy41Nzk2ODc1IDMuMDk0Njg3NSA3LjU3OTY4NzUgMyA5IEMyLjAxIDkuNDk1IDIuMDEgOS40OTUgMSAxMCBDMC42NyA2LjcgMC4zNCAzLjQgMCAwIFogIiBmaWxsPSIjNDEzMDI1IiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgyNzAsMTAwKSIvPgo8cGF0aCBkPSJNMCAwIEMwLjMzIDAgMC42NiAwIDEgMCBDMi4xMjUgNi43NSAyLjEyNSA2Ljc1IDEgOSBDMC4zNCA5IC0wLjMyIDkgLTEgOSBDLTEuMDk3NjU2MjUgMi44NDc2NTYyNSAtMS4wOTc2NTYyNSAyLjg0NzY1NjI1IC0xIDEgQy0wLjY3IDAuNjcgLTAuMzQgMC4zNCAwIDAgWiAiIGZpbGw9IiM0MzMxMkEiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDQzLDE1MikiLz4KPHBhdGggZD0iTTAgMCBDLTAuNjYgMS4zMiAtMS4zMiAyLjY0IC0yIDQgQy0zLjk4IDQgLTUuOTYgNCAtOCA0IEMtNy4zNCAzLjY3IC02LjY4IDMuMzQgLTYgMyBDLTUuNjcgMi4zNCAtNS4zNCAxLjY4IC01IDEgQy0zIDAgLTMgMCAwIDAgWiAiIGZpbGw9IiM0NjIzMTAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDI2OCw0NzcpIi8+CjxwYXRoIGQ9IiIgZmlsbD0iIzI3MDAwMCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMCwwKSIvPgo8cGF0aCBkPSIiIGZpbGw9IiMwMDAwMDAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAsMCkiLz4KPHBhdGggZD0iIiBmaWxsPSIjMDAwMDAwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLDApIi8+CjxwYXRoIGQ9IiIgZmlsbD0iIzAwMDAwMCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMCwwKSIvPgo8cGF0aCBkPSIiIGZpbGw9IiMwMDAwMDAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAsMCkiLz4KPHBhdGggZD0iIiBmaWxsPSIjMDAwMDAwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLDApIi8+CjxwYXRoIGQ9IiIgZmlsbD0iIzAwMDAwMCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMCwwKSIvPgo8cGF0aCBkPSIiIGZpbGw9IiMwMDAwMDAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAsMCkiLz4KPHBhdGggZD0iIiBmaWxsPSIjMDAwMDAwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLDApIi8+CjxwYXRoIGQ9IiIgZmlsbD0iIzAwMDAwMCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMCwwKSIvPgo8cGF0aCBkPSIiIGZpbGw9IiMwMDAwMDAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAsMCkiLz4KPHBhdGggZD0iIiBmaWxsPSIjMEIwQjAwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLDApIi8+CjxwYXRoIGQ9IiIgZmlsbD0iIzAwMDAwMCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMCwwKSIvPgo8cGF0aCBkPSIiIGZpbGw9IiNGRjAwMDAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAsMCkiLz4KPHBhdGggZD0iIiBmaWxsPSIjMDAwMDAwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLDApIi8+CjxwYXRoIGQ9IiIgZmlsbD0iIzAwMDAwMCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMCwwKSIvPgo8cGF0aCBkPSIiIGZpbGw9IiMwMDAwMDAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAsMCkiLz4KPHBhdGggZD0iIiBmaWxsPSIjMDAwMDAwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLDApIi8+CjxwYXRoIGQ9IiIgZmlsbD0iIzAwMDAwMCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMCwwKSIvPgo8cGF0aCBkPSIiIGZpbGw9IiMwMDAwMDAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAsMCkiLz4KPHBhdGggZD0iIiBmaWxsPSIjMDAwMDAwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLDApIi8+Cjwvc3ZnPgo=",
```

Comment:
Can you try to fix the borders?
![image](https://github.com/user-attachments/assets/5b08d0f9-c33b-4b5e-96b4-029cc56bccf0)

Reply:
Dam it! I tried multiple png to svg converter and all had the similar problem. Do you know what the better way is?

Reply:
I also tested it in the common size in the nimiq.watch or nimiqhub, it would not be visible. But if we have a better way to fix it, it's gonna be perfect, but I really have limited graphic editing knowledge.

Reply:
I would say either by hand in tools like figma or use the PNG itself instead of the SVG

Reply:
OK, png logo uploaded and the logo base64 code got updated accordingly as well.

Reply:
No logo path pls, only logo as base64

Reply:
base64 will add the background. It's exactly same as the one in the logo path.

Reply:
logo path removed.